pkg/collector/corechecks/net/network.go

```diff
@@ -199,8 +199,10 @@ func submitInterfaceMetrics(sender aggregator.Sender, interfaceIO net.IOCounters
 	sender.Rate("system.net.bytes_rcvd", float64(interfaceIO.BytesRecv), "", tags)
 	sender.Rate("system.net.bytes_sent", float64(interfaceIO.BytesSent), "", tags)
 	sender.Rate("system.net.packets_in.count", float64(interfaceIO.PacketsRecv), "", tags)
+	sender.Rate("system.net.packets_in.drop", float64(interfaceIO.Dropin), "", tags)
```

Comment:
This will only be used when the Python version of the check is not available (e.g. on IoT or Android environments). You probably want to open a PR on integrations-core for the Python check

Reply:
I've submitted https://github.com/DataDog/integrations-core/pull/10551 which should IIUC complete this PR.